tests/testsuite/conda.rs

```diff
@@ -2,15 +2,10 @@ use ansi_term::Color;
 use std::io;
 
 use crate::common;
-use crate::common::TestCommand;
 
 #[test]
 fn not_in_env() -> io::Result<()> {
-    let output = common::render_module("conda")
-        .env_clear()
-        .env("PATH", env!("PATH"))
-        .use_config("".parse().unwrap())
-        .output()?;
+    let output = common::render_module("conda").output()?;
```

Comment:
Hmm, is this change required? I'm wondering if removing `.env_clear()` will make the development environment messed up with the tests.

Reply:
`.env_clear()` is actually already called by `common::render_module()`.
https://github.com/starship/starship/blob/77c42dca734b9ee537c5c6351d1422bb025b2d30/tests/testsuite/common.rs#L32-L44